### DISCUSSION TITLE
Настоящий Проект Венера

### DISCUSSION THREAD

**Discussion by ivansglazunov** [deep-foundation/dev#28] [Posts]
Мы создаем ресурсоориентированное будущее, открыто и публично разрабатывая ассоциативные программные решения и все под открытой лицензией общественного достояния. Мы не считаем мнения о том, что обществу нужна другая идеология, мораль или система. Любая идеология, мораль и система - следствие имеющихся возможностей, восприятие видимых ограничений и потенциала. Мы верим, что если дать человечеству определенные возможности, общество будет меняться соответственно им, как было со всеми технологиями ранее. Мы верим что футуризм, сказки и непрактическая философия не является шагом к прекрасному будущему, инструкцией к такому шагу или даже мотивацией для него, но они делают голодных до мотивации, мечты и изменений людей сказочниками и псевдофилософами, нежели разработчиками нового будущего. Мы верим что если дать человечеству ассоциативную культуру работы с моделями данных, полную совместимость любых решений, полностью убрать потолок гибкости проектов и повысить реиспользуемость кода с 2-5% сегодня, до хотя бы 85%, тогда акцент будет смещен с "как сделать что бы работало" на "как применить то, что есть" в целях социального благополучия.

[Упомянуть про способы поучаствовать или пожертвовать.]
[Упомянуть что именно есть, и что будет сделано.]
[Упомянуть о культуре организации.]

**Comment by zakhar-kogan** [comment]
Кажется, что это лучше инфографикой было бы описать + в контексте фонда разработки общественного достояния + слоган наподобие _"В мире [больше 25 млн. программистов](https://www.daxx.com/blog/development-trends/number-software-developers-world), и мы верим, что используемые заново 85% кода (Deepcase) вместо 2-5% (сегодня) и применение ассоциативности в данных высвободят сотни тысяч часов в год для умов, которые делают мир лучше каждый день."_

Т.е. что-то в стиле "уверены, что время и нервы разработчиков - самое ценное, и это будет тру реализацией Венеры + ускорит прогресс"

Мб ещё какой-нибудь термин в стиле "OBD - onthology-based development" забабахать? Чтобы было, на что ссылаться + мб поможет за пару очков хайпа (что-то новое) и интереса.
Onthology-driven не покатит, т.к. ODD неблагозвучно; associability-driven тоже нет, т.к. на русском ADD/ADDD.

**Reply by ivansglazunov** [reply]
Время и нервы разработчиков тут не причем. Их тратилось Х и будет тратиться Х. Личный опыт не изменится. Изменение будет заметно существенно не в личном плане. Качественный скачек будет виден только на уровне скорости эволюции бизнеса использующего систему.

**Reply by ivansglazunov** [reply]
ODD/ADD нудно что-то такое, нужно думать

**Reply by ivansglazunov** [reply]
```Кажется, что это лучше инфографикой было бы описать + в контексте фонда разработки общественного достояния + слоган наподобие "В мире больше 25 млн. программистов, и мы верим, что используемые заново 85% кода (Deepcase) вместо 2-5% (сегодня) и применение ассоциативности в данных высвободят сотни тысяч часов в год для умов, которые делают мир лучше каждый день"```
мне очень нравится

**Reply by konard** [reply]
Поддерживаю **ADDD (Associative Data Driven Development)**, это подчёркивает, что это не просто данные собранные без цели связывания и достижения совместимости, а связанные данные, ставящие перед собой цель достигнуть совместимости при минимальных затратах на эту самую совместимость, без необходимости, что-либо переделывать снова и снова, только ради того, чтобы оно давало хоть какой-то результат.
